requirements.txt

```diff
@@ -12,3 +12,5 @@ pytest==3.9.3
 pytest-benchmark>=3.1
 pytest-cov>=2.6.0
 coveralls>=1.5.1
+seaborn>=0.9.0
```

Comment:
this shouldn't be here, as it's not required by the library itself. This should go into "demos" requirements in `setup.py`

Reply:
@jessclmoore Are you sure `seaborn` is required? For which demo?

Reply:
On this demo: ensemble-node-classification-example.ipynb with model_type=’graphsage’ on the Cora dataset, I got the error: ModuleNotFoundError: No module named 'seaborn'

Reply:
Thanks @youph. I've fixed this by removing seaborn from the requirements.txt and pushed back  up.